screens/DirectionsScreen.js

```diff
@@ -37,15 +32,33 @@ const DirectionsScreen = (props) => {
 
     const [address, setAddress] = useState(null)
     const [note, setNote] = useState(null)
-
+    const [client, setClient] = useState(null)
+    const [helpRequestId, setHelpRequestId] = useState(null)
     
     useEffect(() => {
-
-        // TODO: GET the assigned's location and note here
-
-        setLocation(DEFAULT_COORDINATES)
-        convertToAddress(DEFAULT_COORDINATES, setAddress)
-        setNote(DEFAULT_INFORMATION)
+        const currentNotification = props.notification.notificationQueue[0]
+
+        if(!currentNotification) {
+          Alert.alert(
+            "Sorry! Data on this help request cannot be found!",
+            "You will be re-directed back to the main menu",
+            [
+              { text: 'Ok', onPress: () => {
+                Actions.main()
+              }},
+            ],
+            { cancelable: false });
+          return
+        }
+      
+        const { username, location } = currentNotification.user
+        const { coords, note } = location
+
+        setLocation(currentNotification.user.location.coords)
+        convertToAddress(coords, setAddress)
```

Comment:
This is assuming that the user has set a location. What if they didnt? It will have a console error screen to the user if it is not handled.

Reply:
I'm thinking of just popping up an alert and then maybe redirecting off the directions screen?

Reply:
Should we also guide the user to set location if they attempt to start alarm without any lat lon set? perhaps through alerts as well

Reply:
Hmmmmm... The `convertToAddress` here comes from the location received by the helpRequest, so it shouldn't depend on the user's own location

Reply:
yea and the help request lat lon is from the user's lat lon, user meaning the person who initiated the help request

Reply:
Ohh I see what u mean... Yeah I think its smarter to do a prompt on the user's side - if they haven't set a location, they shouldn't be able to send a help request

Reply:
Yes agreed, please make an issue on this :)

Reply:
I can include it in this PR, if everything else is ready to go

Reply:
Sure that works too

Reply:
Check is included in the `using` page!